src/entry/LoadingState.tsx

```diff
@@ -38,7 +38,7 @@ class AcceptTos extends React.Component<any, any> {
                 <span className={classnames(styles.label, styles.client)}>Metamask</span>
               </div>
             </div>
-            <button disabled={!this.state.isChecked} className={classnames(button, styles.grey)}
+            <button disabled={!this.state.isChecked} className={classnames(button, styles.button, greyWhite)}
```

Comment:
here you can use Button component with color="greyWhite"
in Button.tsx add to list of colors the "greyWhite" (line 8)

Reply:
will d